File: csharpServer/model/Destination.cs
namespace model;

public class Destination : Entity<int>
{
    public string Name { get; set; }

    public Destination(): base() { }

    public Destination(int id, string name) :base(id)
    {
        Name = name;
    }
    
    public override string ToString()
    {
        return Name;
    }
}

File: javaClient/model/src/main/java/ro/transportcompany/Destination.java
package ro.transportcompany;

public class Destination extends Entity<Integer> {
    private String name;

    public Destination() {
        super();
    }

    public Destination(Integer id, String name) {
        super(id);
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return name;
    }
}